type-definitions/leaflet-tests.ts

```diff
@@ -0,0 +1,1252 @@
+/// <reference path='leaflet.d.ts' />
```

Comment:
@nathancahill - for when you get back to this pull request: 

See `red flags` section of https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html

It would be nice to have the updated `d.ts` included in the `package.json`